### PR TITLE
Post macro comment for consistency

### PR DESCRIPTION
**Pull request by hvy**
Style fixes for consistency.

### CODE REVIEW

**Comment by niboshi** [comment]
jenkins, test this please

File: chainerx_cc/chainerx/macro.h [hunk 1]
```diff
@@ -40,7 +40,7 @@
 #ifndef CHAINERX_VISIBILITY_HIDDEN
 #if defined(WIN32) || defined(_WIN32)
 #define CHAINERX_VISIBILITY_HIDDEN
-#else
+#else  // CHAINERX_VISIBILITY_HIDDEN
```
[the remark this answers is not in the document]
Comment (hvy) [reply chainerx_cc/chainerx/macro.h]: Thanks for the catch.

**Comment by niboshi** [comment]
Jenkins, test this please

**Comment by niboshi** [comment]
LGTM

**Pull request merged**